Source/src/Gameplay.cpp

```diff
@@ -119,6 +120,21 @@ const Hachiko::ComponentCamera* Hachiko::Debug::GetRenderingCamera()
     return App->camera->GetRenderingCamera();
 }
 
+float Hachiko::Debug::GetFps()
+{
+    return App->renderer->GetCurrentFps();
+}
+
+unsigned int Hachiko::Debug::GetMs()
+{
+    return App->renderer->GetCurrentFps();
```

Comment:
Here we should get current ms instead, but thats something I will change so all good!

Reply:
whoops thanks for noting that, i'm doing a quick fix no problems.

Reply:
Made the necessary change 👍